### ISSUE TITLE
Where is F1_score located in tf.keras

### ISSUE THREAD

**Issue by shashvatshahi1998** [tensorflow/tensorflow#28207]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): TF 1.13
Ubuntu 14.04 LTS
- Are you willing to contribute it (Yes/No): YES

**Describe the feature and the current behavior/state.**
Basically there was an previous issue regarding multi-lable classifier functionalities like(micro,macro) are not present in F1_score calculation. So I tried to add these features but later wards Alextp asked me to add this features in tf.keras as tensorflow is now gong to depend on that for TF2.0, but I am not getting where F1_score is defined in tf.Keras?

**Will this change the current api? How?**
NO

**Who will benefit with this feature?**
All those who wants multi-lable classifiers to be added to F1_score calculations
**Any Other info.**

**Comment by pavithrasv** [comment]
We do not have F1 score metric out of the box in tf.keras. If you think we should add it may be we can add to the AddOns repository (https://github.com/tensorflow/addons) and based on how widely it is being used we can move it to TensorFlow core.

**Comment by shashvatshahi1998** [comment]
Tensorflow has f1_score defined here:
https://github.com/tensorflow/tensorflow/blob/r1.13/tensorflow/contrib/metrics/python/metrics/classification.py
Whats the problem if I contribute modifications there?

**Comment by pavithrasv** [comment]
Contrib module is deprecated in TF 2.0. AddOns repository exists for such use cases so this would be a good fit there.
